application/src/main/java/org/togetherjava/tjbot/commands/moderation/WarnCommand.java

```diff
@@ -22,11 +22,11 @@
 
 
 /**
- * This command can warn users. Warning can also be paired with a warn reason. The command will also
- * try to DM the user to inform them about the action and the reason.
+ * This command can warn users. The command will also try to DM the user to inform them about the
+ * action and the reason.
  * <p>
- * The command fails if the user triggering it is lacking permissions to either ban other users or
- * to ban the specific given user (for example a moderator attempting to ban an admin).
+ * The command fails if the user triggering it is lacking permissions to either warn other users or
+ * to warn the specific given user (for example a moderator attempting to ban an admin).
```

Comment:
there is still a "ban" in the last sentence. "(for example a moderator attempting to ban an admin)"